src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,156 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Duke {
+    protected static final String HR = "    ____________________________________________________________\n";
+    protected static final String GREET = "     Hello! I'm Duke\n" +
+            "     What can I do for you?\n";
+    protected static final String BYE = "     Bye. Hope to see you again soon!\n";
+    protected static final String ECHO = "     %s\n";
+    protected static final String ADD = "     Got it. I've added this task: \n" +
+            "       %s\n" +
+            "     Now you have %d tasks in the list.\n";
+    protected static final String TASK = "     %d. %s\n";
+    protected static final String DONE = "     Nice! I've marked this task as done: \n" +
+            "       %s\n";
+
+    protected static ArrayList<Task> tasks = new ArrayList<>(100);
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String stdIn = "greet";
+        String stdOut = "";
+
+        do {
+            try {
+                stdOut = parseCmd(stdIn);
+            } catch (UnknownArgumentException e) {
+                stdOut = e.toString();
+            } catch (EmptyArgumentException e) {
+                stdOut = e.toString();
+            } catch (NumberFormatException e) {
+                stdOut = e.toString();
+            } catch (IndexOutOfBoundsException e) {
+                stdOut = e.toString();
+            } catch (Exception e) {
+                stdOut = e.toString();
+            } finally {
+                System.out.println(HR + stdOut + HR);
+            }
+
+            stdIn = scan.nextLine();
+        } while (!stdIn.equals("bye"));
+
+        System.out.println(BYE);
+    }
+
+    protected static String parseCmd(String line) throws EmptyArgumentException, UnknownArgumentException {
+        // Take commands from first space-separated word
+        String[] cmd = line.split(" ");
+        if (cmd.length == 0) {
+            throw new EmptyArgumentException("command");
+        }
+
+        switch (cmd[0]) {
```

Comment:
I noticed that your `case` within your switch `block` is enclosed with {}. This is not necessary. Also, you may want to consider putting the `break` keyword within each `case` block. You can refer to this portion of the Java Code Style.

![Screenshot 2020-09-15 at 12 18 52 AM](https://user-images.githubusercontent.com/59989652/93111353-12b6bc80-f6e9-11ea-97d6-6611b8c775e9.png)